CoreUtilities/src/au/gov/asd/tac/constellation/utilities/camera/BoundingBox.java

```diff
@@ -341,6 +341,28 @@ public void set(final Vector3f min, final Vector3f max, final Vector3f min2, fin
 
         isEmpty = false;
     }
+    
+    /**
+     * Method used for testing to check if BoundingBox values are equal
+     * 
+     * @param bbox the BoundingBox to compare to this instance
+     * @return true if the BoundingBox are the same, false otherwise
+     */
+    public boolean areSame(final BoundingBox bbox) {
+        if(min.areSame(bbox.getMin())){
+            return false;
+        }
+        if(min2.areSame(bbox.getMin2())){
+            return false;
+        }
+        if(max.areSame(bbox.getMax())){
+            return false;
+        }
+        if(max2.areSame(bbox.getMax2())){
+            return false;
+        }
+        return true && Boolean.compare(isEmpty, bbox.isEmpty()) == 0;
```

Comment:
combine all of these into one boolean return (also `true && blah` is the same as `blah`):
`return !min.areSame(bbox.getMin()) && ... !max2.areSame(bbox.getMax2()) && Boolean.compare(isEmpty, bbox.isEmpty()) == 0`